docs/data/base/components/button/UnstyledButtonIntroduction/tailwind/index.tsx

```diff
@@ -0,0 +1,19 @@
+import * as React from 'react';
+import Button from '@mui/base/Button';
+import Stack from '@mui/material/Stack';
+
+export default function UnstyledButtonsSimple() {
+  return (
+    <Stack spacing={2} direction="row">
+      <Button className="cursor-pointer disabled:cursor-not-allowed text-sm bg-violet-500 hover:bg-violet-600 active:bg-violet-700 text-white rounded-md font-semibold px-4 py-2 border-none disabled:opacity-50">
+        Button
+      </Button>
+      <Button
+        className="cursor-pointer disabled:cursor-not-allowed text-sm bg-violet-500 text-white rounded-md font-semibold px-4 py-2 border-none disabled:opacity-50"
```

Comment:
```suggestion
      <Button
        className="cursor-pointer disabled:cursor-not-allowed text-sm bg-violet-500 text-white rounded-md font-semibold px-4 py-2 border-none disabled:opacity-50"
```

We discussed that it would probably be better if we create a `CustomButton` instead of adding all classes inline in each button's instance. For e.g.

```
const CustomButton = React.forwardRef((props, ref) => {
  const { className, ...other } = props;
  return (
    <Button className={clsx("cursor-pointer disabled:cursor-not-allowed text-sm bg-violet-500 text-white rounded-md font-semibold px-4 py-2 border-none disabled:opacity-50", className)} {...other} ref={ref} />
  )
});
```

I would propose to update all demos following this pattern. It's closer to how developers would do it in their project.

Reply:
Thanks @mnajdova for the explanation. I pushed the changes that you suggested. I'll continue to work on other demos and components if you agree with the implementation :)